interface/src/app/gene/network/network.js

```diff
@@ -0,0 +1,262 @@
+/**
+ * "adage.gene.network" module.
+ */
+
+angular.module('adage.gene.network', [
+  'ui.router',
+  'placeholders',
+  'ui.bootstrap',
+  'ui.slider'
+])
+
+.config(function config($stateProvider) {
+  $stateProvider.state('gene_network', {
+    url: '/gene_network/?genes&mlmodel',
+    views: {
+      main: {
+        templateUrl: 'gene/network/network.tpl.html',
+        controller: 'GeneNetworkCtrl',
+        controllerAs: 'ctrl'
```

Comment:
I like it because the code is easier to follow and we don't need $scope as a dependency any more.  We probably should create an issue to change to this syntax in the long run.